main.js

```diff
@@ -97,6 +101,30 @@ jQuery(function($) {
 
     // detect if we cross 992px window width.
     window.matchMedia('(min-width: 992px)').addListener(cleanup);
+
+    // disable active language in language menu (full width)
+    $('.js__header-top__language-navigation.fixed a.active').on('click', function(e) {
+        e.preventDefault();
+    });
+
+    // toggle language menu (dropdown)
+    $('.js__language-navigation__toggle-btn').on('click', function(e) {
+        e.preventDefault();
+        $('.js__language-navigation__items-wrp').not(':animated').slideToggle(300);
```

Comment:
@yosokus Please collect number values as settings in centralized place above. Toggle Speed, min width etc. Now the min width value is repeated in the code.